HW3.py

```diff
@@ -0,0 +1,116 @@
+# Created on December 26, 2019
+
+
+import sys
+import re
+import operator
+
+NO_GUESSES = "I don't know any word of a given length..."
+ONE_MORE_TRY = 'Would you like to play one more times? (y/n): '
+YES = 'Y'
+BYE = 'Bye, see you next time! :)'
+WELCOME ='Welcome to a Guessing Word Game!'
+THINK_A_WORD = "Think of a word and enter it's length: "
+POSITIONS_OF_AVAILABLE_LETTERS ='\n If letter(s): "{}" is available in the hidden word, \n please enter comma separated position(s) counting from 0 like 0,2, \n or just press Enter if the letter is absent: '
+EMPTY_STRING = ''
+COMMA = ','
+DOT = '.'
+GUESSED_WORD = 'I want to tell wholw word: "{}"'
+CANT_GUESS = "You are che champions! Can't guess the word matching specified letters"
+NO_TRIES_LEFT = "No more word, so you're probably cheating, stop doing it!"
+PATH_TO_DATABASE = 'C:\Python\words.txt'
```

Comment:
The absolute path using isn't good practice. It'd be better to use relative paths. 
Also, I'd change the variable name to PATH_TO_FILE, not to DATABASE.